ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml

```diff
@@ -164,8 +165,15 @@
     }
     <div class="@divLength wide stretched column">
         <div class="lh-content" id="lh-comments">
-            @await Html.PartialAsync("Partials/CommentsPartial", new ViewDataDictionary(ViewData.WithLang(language).WithTime(timeZone))
-                   { {"PageOwner", Model.ProfileUser.UserId}, })
+            @if (Model.ProfileUser.IsBanned)
+            {
+                <p>Comments are disabled because the user is banned.</p>
```

Comment:
```suggestion
                <div class="ui yellow segment" id="comments">
                    <p>Comments are disabled because the user is banned.</p>
                </div>
```
This would look better if you used the same container style as the original comments